### PR TITLE
ENG-8823: Adds note to specify quota for all resources even if zero

### PR DESCRIPTION
**Pull request by bredamc**
ENG-8823: Adds note to clarify that all resources must have quota specified even if value is 0

### CODE REVIEW

**Comment by KPostOffice** [comment]
This looks good to me. Do we need to mention the actual name of the covered resource that a user should put in the covered resources as well? In this case it would be `nvidia.com/gpu`

**Review by KPostOffice — COMMENTED** [review]
This looks good to me. Do we need to mention the actual name of the covered resource that a user should put in the covered resources as well? In this case it would be nvidia.com/gpu

**Comment by bredamc** [comment]
Thanks @KPostOffice. Updated based on our chat in DM.

File: modules/configuring-quota-management-for-distributed-workloads.adoc [hunk 1]
```diff
@@ -104,7 +104,12 @@ The cluster queue will start a distributed workload only if the total required r
...
 [NOTE]
 ====
-In this release of {productname-short}, the only accelerators supported for distributed workloads are NVIDIA GPUs.
+. Specify a quota for each resource that the user can request (even if the requested value is 0), by updating the `spec.resourceGroups` section as follows:
+
+* Include the resource name in the `coveredResources` list.
+* Specify the resource `name` and `nominalQuota` in the `flavors.resources` section, even if the `nominalQuota` value is 0.
+
+. In this release of {productname-short}, the only accelerators supported for distributed workloads are NVIDIA GPUs.
 ====
```
Comment (MelissaFlinn) [inline modules/configuring-quota-management-for-distributed-workloads.adoc:113]: Is it correct that lines 107-111 should be within the note?  Seems to be a step

Comment (bredamc) [reply modules/configuring-quota-management-for-distributed-workloads.adoc]: Good point! The update started out as a simpler note, but you're right that it's now better included as part of the step. Updated accordingly, thank you!

**Comment by MelissaFlinn** [comment]
lgtm

**Pull request merged**